.travis.yml

```diff
@@ -32,6 +32,21 @@ matrix:
       osx_image: xcode7.3
       env: SCALANATIVE_GC=immix
 
+addons:
+  apt:
+    sources:
+    - ubuntu-toolchain-r-test
+    - llvm-toolchain-precise-3.7
+    packages:
+    - clang++-3.7
+    - llvm-3.7
+    - llvm-3.7-dev
+    - llvm-3.7-runtime
+    - llvm-3.7-tools
+    - libgc-dev
+    - libunwind7-dev
+    - make
+
```

Comment:
There's quite a few things to fix before the travis setup script can be used, such as making the LLVM version configurable. I've removed the `addons: apt` syntax, not sure if `cache: apt` will still work. It should if it caches the `.deb` files.